CHANGELOG.md

```diff
@@ -12,6 +12,9 @@
 #### Changed
 - Upgraded to IDE Rider v3.0.28 (#166)
 
+#### Fixed
+- Reset values and buffs after respawn and added local pooling for explosionParticles to optimize performance and showcase built-in pooling component (#167)
```

Comment:
1. These should be on 2 different lines, as they're 2 different changes
2. The first is a fix, but the second isn't. It should be in the "Changed" or "Added" categories.